components/network/quagga/patches/70-isisd-trill.patch

```diff
@@ -0,0 +1,8348 @@
+diff --git configure.ac configure.ac
```

Comment:
What is it for?? I suppose, carrying around 8000-loc patch should be justified.

Reply:
Sun created this "patch" which introduces TRILL and ISIS support into quagga which is not there in the standard sources. It's a Sun enhancement and has been there for ages, even in the Sol9 SFW packages. Users might expect that the features will also be available on OI.
Yes, if someone has enough time, it would be cool to bring these patches upstream into quagga...